### PR TITLE
Fix pacloc

### PR DESCRIPTION
**Pull request by karllark**
Update to handle the new location of the pack files.  Should fix the test failure as well as mean the doc example works again.

Includes a minor update to the plotting of the model spectrum.  Model wavelength points made in log space instead of linear space.  Better matches wide wavelength ranges and a constant resolution.

### CODE REVIEW

**Review by jdtsmith — COMMENTED** [review]
Looks good; log spacing is a good idea.  @Ameek-Sidhu  and @drvdputt, be sure to re-merge from master as you work to port helpers stuff into the base object.

File: setup.cfg [hunk 1]
```diff
@@ -29,7 +29,7 @@ docs =
     sphinx-astropy
 
 [options.package_data]
-pahfit = data/*, packs/*
+pahfit = data/*, packs/*, packs/science/*, packs/instrument/*
```
[the remark this answers is not in the document]
Comment (karllark) [reply setup.cfg:32]: There is a yml file there now.  Will this stay?  We can get more specific on what to include later as well.

[the remark this answers is not in the document]
Comment (jdtsmith) [reply setup.cfg:32]: That vestigial, an _example_ from 3.5 years ago (!) that doesn't have the correct syntax.  I'm OK with just deleting it as part of this PR.  I had removed another such old example, but forgot this one.

[the remark this answers is not in the document]
Comment (karllark) [reply setup.cfg]: Ok.  Done.

**Review by jdtsmith — APPROVED** [review]
Looks good thanks.

**Pull request merged**